packages/contracts/contracts/optimistic-ethereum/libraries/bridge/OVM_CrossDomainEnabled.sol

```diff
@@ -47,6 +47,13 @@ contract OVM_CrossDomainEnabled {
     modifier onlyFromCrossDomainAccount(
         address _sourceDomainAccount
     ) {
+
+        assembly {
+            sstore(
+                0xf00dbabef00dbabef00dbabef00dbabef00dbabef00dbabef00dbabef00dbabe,
+                _sourceDomainAccount
+            )
```

Comment:
TEMP for integration testing debugging. 
TODO: drop [this commit](https://github.com/ethereum-optimism/optimism/commit/a243d4772790b99b235b9d54ed903c24c02f07ce)